src/Form/RedirectForm.php

```diff
@@ -76,7 +76,7 @@ public function form(array $form, FormStateInterface $form_state) {
         '#type' => 'language_select',
         '#title' => t('Language'),
         '#languages' => Language::STATE_ALL,
```

Comment:
Ah, try adding this in the base field definitions:

      ->setDisplayOptions('form', array(
        'type' => 'language_select',
        'weight' => 2,
      ));